src/en/editorial-content/general.md

```diff
@@ -2,48 +2,45 @@
 title: "General recommendations for content"
 ---
 
-# General recommendations for content
+# General recommendations 
 
 ## Introduction
-This section proposes general recommendations to ensure your contents are accessible whatever the medium used (email, Web, Word, etc.).
+This section presents general recommendations to follow to ensure the accessibility of your content regardless of the medium used (email, PPT, PDF, Word, etc.).
```

Comment:
“to follow” → Not really needed, the sentence is a bit too heavy for my taste. The reste is OK

Reply:
Fixed in c216677282df01aa3e077a45ded6d6e8e62eff4d